docs/dev/openstack/customization.md

```diff
@@ -13,6 +13,15 @@ export OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE="https://example.com/my-rhcos.qcow2"
 
 **NOTE:** For this to work, the environment variable value must be a valid http(s) URL.
 
+If the user wants to upload the image from the local file system, he can set the environment variable value as `file:///path/to/file`. In this case the installer will take this file and automatically create an image in Glance.
+
+Example:
+
+```sh
+export OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE="file:///home/user/rhcos.qcow2"
```

Comment:
Please don't document the env. It's only a development tool.

Reply:
This is for developers documentation... Since we have the variable documented in `docs/dev`, I think it is reasonable to add "file://" part there as well.